lib/beaker-pe/install/pe_utils.rb

```diff
@@ -306,42 +306,45 @@ def deploy_frictionless_to_master(host)
           if platform == "aix-7.2-power"
             platform = "aix-7.1-power"
           end
-          klass = platform.gsub(/-/, '_').gsub(/\./,'')
-          if host['platform'] =~ /windows/
-            if host['template'] =~ /i386/
-              klass = "pe_repo::platform::windows_i386"
+
+          if ! use_meep_for_classification?(master[:pe_ver], options)
```

Comment:
Rather than wrap the block which causes a lot of whitespace damage, let's either exit the function early or not call the function in the first place, and update the yard docs.